doc/introduction/pulse.rst

```diff
@@ -0,0 +1,289 @@
+can be found using the :func:`~qml.evolve` function:
+
+.. code-block:: python
+
+    from jax import numpy as jnp
+
+    f1 = lambda p, t: p * jnp.sin(t) * (t - 1)
+    H = 2 * qml.PauliX(0) + f1 * qml.PauliY(1)
+    ev = qml.evolve(H)
+    ev
+    >>> ParametrizedEvolution(wires=[0, 1])
+
+The initial :class:`~.ParametrizedEvolution` does not have parameters defined, and so will
+not have a defined matrix. To complete initialization of the :class:`~.Operator`, it must be passed
+parameters and a time interval:
+
+.. code-block:: python
+
+    ev([1.2], t=[0, 4]).matrix()
+    >>> Array([[-0.14115842+0.j        ,  0.03528605+0.j        ,
+                 0.        -0.95982337j,  0.        +0.23993255j],
+               [-0.03528605+0.j        , -0.14115842+0.j        ,
+                 0.        -0.23993255j,  0.        -0.95982337j],
+               [ 0.        -0.95982337j,  0.        +0.23993255j,
+                -0.14115842+0.j        ,  0.03528605+0.j        ],
+               [ 0.        -0.23993255j,  0.        -0.95982337j,
+                -0.03528605+0.j        , -0.14115842+0.j        ]],      dtype=complex64)
+
+
+The parameters can be updated by calling the :class:`~.ParametrizedEvolution` again with different inputs.
+
+Additional options with regards to how the matrix is calculated can be passed to the :class:`.ParametrizedEvolution`
+along with the parameters, as keyword arguments:
+
+.. code-block:: python
+
+    qml.evolve(H)(params=[1.2], t=[0, 4], atol=1e-6, mxstep=1)
+
+These options are:
+
+- `atol (float, optional)`: Absolute error tolerance
+- `rtol (float, optional)`: Relative error tolerance
+- `mxstep (int, optional)`: maximum number of steps to take for each time point
+- `hmax (float, optional)`: maximum step size allowed
+
+If not specified, they will default to predetermined values. See :class:`.ParametrizedEvolution` for details.
+
+Using qml.evolve in a QNode
+^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+The :class:`~.ParametrizedEvolution` can be implemented in the QNode in the same way as any other
+PennyLane :class:`~.Operator`.
+
+To look at an example of this, let's start with two instances of :class:`~.ParametrizedHamiltonian`:
+
+.. code-block:: python
+
+    ops = [qml.PauliX(0), qml.PauliY(1), qml.PauliZ(2)]
+    coeffs = [lambda p, t: p for _ in range(3)]
+    H1 = qml.ops.dot(coeffs, ops)  # time-independent parametrized hamiltonian
+
+.. code-block:: python
+
+    ops = [qml.PauliZ(0), qml.PauliY(1), qml.PauliX(2)]
+    coeffs = [lambda p, t: p * jnp.sin(t) for _ in range(3)]
+    H2 = qml.ops.dot(coeffs, ops) # time-dependent parametrized hamiltonian
+
+Now we can execute the evolution of these Hamiltonians applied simultaneously:
+
+.. code-block:: python
+
+    dev = qml.device("default.qubit", wires=3)
+    @qml.qnode(dev, interface="jax")
+    def circuit(params):
+        qml.evolve(H1 + H2)(params, t=[0, 10])
```

Comment:
Wait what? :exploding_head: I wasn't aware we can simply add `ParametrizedHamiltonian` instances! That's cool!
The order of the passed parameters is just the order of addition?
Maybe this deserves a comment, both because it is a very cool feature, and because there might be tiny gotchas?

Reply:
Yes, if `H1` has params `[1, 2, 3]` and `H2` has params `[4, [5, 6]]`, then `H1+H2` would take params `[1, 2, 3, 4, [5, 6]]` (and `H2 + H1` would take `[4, [5, 6], 1, 2, 3]`). I'll add a comment on this.

The only gotcha I can come up with is that the resulting wire order for `H1+H2` might not match people's expectations if H1 and H2 contain both fixed and parametrized terms (it behaves as if the fixed terms are moved to be together at the beginning, i.e. `fixed1+fixed2+parametrized1+parametrized2`  instead of `fixed1+parametrized1+fixed2+parametrized2`, and then gets the wire order from that). That might be too specific/niche to comment on here?

Reply:
Oh, that is a gotcha indeed :sweat_smile: I'm not sure where we should put this information, but it seems like a useful information if one wants to make use of this blackboardstyle notation...